server/index.js

```diff
@@ -57,4 +62,9 @@ app.post('/login', function (req, res) {
     res.send('login successful');
 });
 
+app.post('/logout', async (req, res) => {
```

Comment:
prefer to use `app.get()` here. I don't recommend to use `res.redirect` with `post`, could cause some trouble, like AJAX `post` would not redirect to target page.